AllReadyApp/Web-App/AllReady.UnitTest/Controllers/OrganizationControllerTests.cs

```diff
@@ -52,6 +55,76 @@ public void ShowOrganization_ReturnsNotFoundForNullOrganization()
             Assert.NotNull(result);
         }
 
+        [Fact]
+        public void ShowOrganization_ReturnsNotFoundForZeroId()
+        {
+            OrganizationController controller;
+            var mockMediator = MockMediatorOrganizationDetailsQuery(out controller);
+
+            var result = controller.ShowOrganization(0).Result as HttpNotFoundResult;
```

Comment:
`Task.Result` will cause deadlocks in AppVeyor. Change the signature of the test to `async Task` and `await` the async function.